backup.sh

```diff
@@ -30,6 +30,7 @@ else
     echo "Backup Failed with Status ${rc}"
     restic unlock
     copyErrorLog
+    kill 1
```

Comment:
@Niondir I haven't been able yet to see if this does what I intended...
Technically, it could.
Not sure if it should.
Even less so if it does.

The rationale, anyhow:
I have a scheduler and want to be reassured as long as the service is green, once I see red in the dashboard, I want to know something is to be looked after. Red, in this context, would be "instance down"

Reply:
I agree, since you could still set your container to "restart: always" to work arround but at least can see the restart timestamp in "docker ps".